### PR TITLE
Bookstore: Use Redux in React components

### PR DESCRIPTION
**Pull request by Tracy-miranja**
- [ ] Wrapped my <App> with the <Provider> component from react-redux.
- [ ] Passed my configured store into the <Provider>
- [ ] Created an initialState variable for your booksSlice.
- [ ] Displayed  my books, received from the slice, in a reusable component.
- [ ] Dispatched actions using useDispatch.
- [ ] Add a <Button> component, which includes:
   -An event handler that adds a book to the books array (with attributes id, title and author).
- [ ] Add a <Button> component, which includes:
   -An event handler that removes a book from the books array (by id)

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function Nav() {
       <h1>BOOKSTORE CMS</h1>
       <ul>
         <li>
-          <Link to="/">BOOKS</Link>
+          <Link to="/form">BOOKS</Link>
```
Comment (julie-ify) [inline src/components/Navbar.js:10]: - [ ] **[OPTIONAL]** I think you should maintain the root path instead of `/form`

File: src/App.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 import Nav from './components/Navbar';
+import AddBookForm from './components/form';
 
 function App() {
   return (
     <>
       <Nav />
       <Routes>
-        <Route exact path="/" element={<Books />} />
+        <Route path="/form" element={<AddBookForm />} />
```
Comment (julie-ify) [inline src/App.js:11]: - [ ] **[OPTIONAL]** I think you should maintain the initial root path for books you had. `/form` seems a bit off.

**Review by julie-ify — APPROVED** [review]
Hi @Tracy-miranja,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (Tracy-miranja) [reply src/App.js:11]: thanks for the feedback @ [julie-ify](https://github.com/julie-ify), ill make the optional changes

**Pull request merged**